docs/cvl/builtin.md

```diff
@@ -36,11 +60,11 @@ The basic sanity rule is limited. This is because it only requires finding a _si
 For example, consider:
```

Comment:
I think it is helpful to include the actual error message so that people can google an error message and find the right documentation

Reply:
Although if the error message says something like "you must enable --multi_assert_check to use builtin rule deepSanity" then you can probably just remove the note about the error message.